### ISSUE TITLE
Tax Treatment Variation for Winnings

### ISSUE THREAD

**Issue by anjoung** [huntallcott/Lotteries#17]
The goal of this issue is to write a memo detailing any documented changes in how lottery winnings have been taxed over the last 20-years. I'll read the news to see if there is any reporting on changes. Additionally, I will be looking into historic data on top marginal tax rates at the federal and state level. I will also compile information on how lottery winnings are reported for tax purposes (as income? capital gains? etc.)

**Comment by anjoung** [comment]
Top marginal tax rates at the federal level are recorded [here](https://www.taxpolicycenter.org/statistics/historical-highest-marginal-income-tax-rates) by the Tax Policy Center. Additionally, [here](https://www.taxpolicycenter.org/statistics/state-individual-income-tax-rates-2000-2018) are the top marginal tax rates at the state level from the Tax Policy Center.

A recent paper by Mertens and Olea (2018) in QJE has constructed a time series from 1946 to 2012 on average marginal tax rates. In Figure 1 of their paper, they chart out this time series for the top rates. The series are based on the marginal federal individual income tax rates and social security contribution rates. They do not include state taxes because they believe that the amount of variation in state-level taxes is very small (Barro and Redlick 2011). The data used to construct this figure is [here](https://github.com/huntallcott/Lotteries/files/2699948/replication_files_for_QJE.zip). 

I have made a new folder called "tax treatment variation" inside the in the DataCodebooks that contains the data that I linked to.
